### DISCUSSION TITLE
How to compile the code?

### DISCUSSION THREAD

**Discussion by luo1368666** [etorth/mir2x#27] [Q&A]
Can you provide a set of compiled engines? Or version, I don't know how to compile it into a finished engine, and I don't know where to download the server and client. I hope to get help, thank you very much.

**Comment by wealthcom996** [comment]
![Uploading nn6.png…]()
![Uploading nn6.png…]()
![Uploading nn6.png…]()
![Uploading nn6.png…]()
2026/06/12 18:01:58 006949	FATAL [main.cpp->int main(int, char**):107]	
	*******	 EXIT trigger caused by LOG(FATAL) entry: 
	"Caught exception: In file: xmltypeset.cpp:583, function: void XMLTypeset::checkDefaultFontEx() const: invalid default font: font = 11, fontsize = 15
*******	STACKDUMP *******
	stack dump [1]  ./client(+0x16e8b7d) [0x55f48faceb7d]  什么原因 客户端checkDefaultFontEx() 无效字体
